acos_client/v21/action.py

```diff
@@ -35,3 +35,13 @@ def reload(self, write_memory=False, **kwargs):
         write_memory = 1 if write_memory else 0
         return self._post("system.action.reload",
                           params={"write_memory": write_memory}, **kwargs)
+
+    def write_active(self, shared_partition=None, tenant_partition=None, **kwargs):
```

Comment:
This should just take one arg, "partition". If you need higher-level logic, like shared+tenant, that should happen up above.